style.css

```diff
@@ -398,3 +444,172 @@ textarea:focus {
 .contact-me-form-button {
   align-self: flex-start;
 }
+
+/* Desktop version of site */
+@media screen and (min-width: 768px) {
+  .logo > a {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+    margin: 0 0 0 100px;
+    font-size: 18px;
+  }
+
+  .headline-main {
+    border-radius: 0 0 0 4em;
+    background-color: #fff;
+    background-image: url(./assets/Header\ bg.png);
+    background-size: 96%;
+    background-position-x: center;
+    background-repeat: no-repeat;
+    height: 100vh;
+  }
+
+  .headline {
+    margin: 10px 0;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    width: 751px;
+    height: 268px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .nav-links {
+    position: absolute;
+    top: 8px;
+    right: 120px;
+    display: flex;
+    margin-top: 15px;
+    padding: 0;
+    color: #6070ff;
+    font-size: 18px;
+  }
+
+  .nav-link {
+    margin: 0 10px;
+    cursor: pointer;
+    color: #344563;
+    font-size: 15px;
+  }
+
+  .social-icons {
+    margin-block: 10px;
+  }
+
+  header {
+    padding: 23px;
+    padding-bottom: 75px;
+  }
+
+  .headline-heading {
+    align-items: center;
+    color: #172b4d;
+  }
+
+  .works-card > img {
+    width: 100%;
+    height: 400px;
+  }
+
+  .works-card:nth-child(even) > img {
+    order: 2;
+  }
+
+  .works-card:nth-child(even) > .works-card-main {
+    order: 1;
+  }
+
+  .works-card {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 2.5em;
+    padding: 1em;
+  }
+
+  .headline-skill-container {
+    display: flex;
+    flex-direction: row;
+    height: 590px;
+  }
+
+  .about-headline {
+    margin: 10px 0;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    height: 100%;
+    width: 50%;
+  }
+
+  .heading-large {
+    font-size: 40px;
+  }
+
+  .about .headline-sm-suport {
+    margin-left: 0;
+  }
+
+  .skills-1 {
+    margin-top: -28px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .about-skill-body-container {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+  }
+
+  .about-skill-body {
+    flex-direction: column;
+    height: 120px;
+    margin-top: 36px;
+    padding-top: 2%;
+    padding-left: 2%;
+    width: 31%;
+    align-items: flex-start;
+  }
+
+  .about-skill-body-img {
+    width: fit-content;
+  }
+
+  .social {
+    position: absolute;
+    left: 42px;
+    margin-top: 190px;
+  }
+
+  .about-skills-container {
+    width: 50%;
+  }
+
+  .contact-me-form-container {
+    background-color: #6070ff;
+    background-image: url(./assets/contact\ form\ background\ shapes.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    align-content: center;
+    padding: 10% 25%;
+    margin-top: 0;
+  }
```

Comment:
- KIndly, minimize the your contact form soze to stick the Figma design and have better UI 👍 
### Your Design 👇
<img width="921" alt="Screenshot 2023-06-14 200141" src="https://github.com/Afsheen96/Portfolio/assets/52010789/8ea0eb92-9a2e-407a-a700-07bf3bc1c225">

### Figma Design 👇
<img width="890" alt="Screenshot 2023-06-14 200011" src="https://github.com/Afsheen96/Portfolio/assets/52010789/d5c17bb3-0635-4135-b6c5-ad081e59bdb9">